### PR TITLE
0.6.0

### PR DESCRIPTION
**Pull request by spvagelis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Bespot iOS SDK offers proximity events and analytics reporting to 3rd party apps
 ## Requirements
 
 - iOS 12.0+
```
Comment (cjd1884) [inline README.md:18]: @spvagelis  Is this still the case or we now run iOS 15.0+?

Comment (spvagelis) [reply README.md:18]: The current version of Beengo is iOS 15, so it could be changed to iOS 15.

Comment (cjd1884) [reply README.md:18]: I was referring to the SDK. I suppose we did not updated the deployment target to 15+. So we are OK.

**Review by cjd1884 — APPROVED** [review]
@spvagelis Very good job! Well done!

**Pull request merged**